src/rgw/rgw_auth.h

```diff
@@ -455,7 +455,7 @@ class LocalApplier : public IdentityApplier {
 
   LocalApplier(CephContext* const cct,
                const RGWUserInfo& user_info,
-               std::string subuser)
+               const std::string &subuser)
```

Comment:
it looks like this one was intentional, because it does `subuser(std::move(subuser))`. i've spotted a few other places in rgw that use this pattern, and i'd prefer to keep them that way

Reply:
Got it. I'll remove the const references on the remaining ones too.